### PR TITLE
architecture & basic design

### PR DESCRIPTION
**Pull request by rahulxyz**
1. Code Architecture updated
2. Absolute path config
3. Bootstrap 
4. Navbar
5. Footer
6. Banner section 
7. Responsive Design

### CODE REVIEW

**Comment by sjha2048** [comment]
Hey @rahulxyz , I am not sure about this, but in the mobile view, the footer is supposed to move dynamically with the banner image?
image
image

Rest this basic layout looks good to me.

**Comment by rahulxyz** [comment]
Banner and footer are independent of each other. Currently, the content in the footer is at the center since there isn't much content. And to maintain responsiveness the content of the entire page is aligned within the same boundary (and at center ), only background color stretches after a certain width.

**Comment by sjha2048** [comment]
Got it

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+import 'bootstrap/dist/css/bootstrap.css';
+import "jquery/dist/jquery.min.js";
+import "popper.js/dist/umd/popper.min.js";
+import "bootstrap/dist/js/bootstrap.min.js";
+import './index.css';
+
```
Comment (NidhiKJha) [inline src/index.js:11]: I am still not sure if bootstrap is the perfect third party library to use here. I would recommend you to write the css from scratch. Would be really great if you can move ahead with SASS.

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+    <App />,
   document.getElementById('root')
```
Comment (NidhiKJha) [inline src/index.js:14]: Why did you remove `StrictMode`

**Comment by NidhiKJha** [comment]
Hi @rahulxyz, Before making such big changes I would appreciate you to come up with design mock ups. There are some features like the dark mode that I have added intentionally.Let me know if you need any help

**Comment by rahulxyz** [comment]
I'm aware that these are major changes. But I noticed that you had just started with the boilerplate code and it would be better if we followed a better architecture at the start. 
As far as I'm aware of there is no design. We have only a basic idea of what we want to build. I'm using bootstrap because of two reasons:
1. We do not have a specific design to follow. So it will be easier to utilize its prebuilt features
2. Currently, we are not focused on building a website where we need to ensure small build size, so it seems to be a viable choice.
As for theme transition, it can be handled even in this code as well
I'm not sure if React.Strict mode is necessary. Is there any particular reason you're using?

**Comment by NidhiKJha** [comment]
Thanks for the pull request. I can't see any comments resolved for now. So, I am closing it for now.
